external/slowparse/slowparse.js

```diff
@@ -1211,6 +1222,12 @@
         throw new ParseError("MISSING_CSS_VALUE", this, this.stream.pos-1, this.stream.pos);
       }
 
+      // Look for missing ; at the end of properties
+      if (value.indexOf('\n') > -1) {
```

Comment:
Could you add a comment as to why checking for '\n' indicates that we're missing a ';'?